### ISSUE TITLE
WeatherIntegrationTest fails with 401 Unauthorized

### ISSUE THREAD

**Issue by tdiesler** [wildfly-extras/wildfly-camel#1195]
This may be a betamax issue - shows on branch [2.18.x](https://github.com/wildfly-extras/wildfly-camel/tree/2.18.x)

```
Caused by: java.lang.IllegalStateException: Got the invalid http status value 'HTTP/1.1 401 Unauthorized' as the result of the query 'http://api.openweathermap.org/data/2.5/forecast/daily?q=Madrid,Spain&lang=en&cnt=7'
	at org.apache.camel.component.weather.WeatherProducer.process(WeatherProducer.java:54)
	at org.apache.camel.util.AsyncProcessorConverterHelper$ProcessorToAsyncProcessorBridge.process(AsyncProcessorConverterHelper.java:61)
	at org.apache.camel.processor.CamelInternalProcessor.process(CamelInternalProcessor.java:190)
	at org.apache.camel.util.AsyncProcessorHelper.process(AsyncProcessorHelper.java:109)
	at org.apache.camel.processor.DelegateAsyncProcessor.process(DelegateAsyncProcessor.java:87)
	at org.apache.camel.impl.ProducerCache$2.doInProducer(ProducerCache.java:447)
	at org.apache.camel.impl.ProducerCache$2.doInProducer(ProducerCache.java:401)
	at org.apache.camel.impl.ProducerCache.doInProducer(ProducerCache.java:291)
	at org.apache.camel.impl.ProducerCache.sendExchange(ProducerCache.java:401)
	at org.apache.camel.impl.ProducerCache.send(ProducerCache.java:241)
	at org.apache.camel.impl.DefaultProducerTemplate.send(DefaultProducerTemplate.java:135)
	at org.apache.camel.impl.DefaultProducerTemplate.sendBody(DefaultProducerTemplate.java:143)
```

**Comment by jamesnetherton** [comment]
No, the service requires an API key now

**Comment by jamesnetherton** [comment]
camel-weather has moved to using HttpClient, which by default does not respect the Java system properties for HTTP proxy servers (betamax sets up proxy server to capture traffic).

https://issues.apache.org/jira/browse/CAMEL-9273

**Comment by jamesnetherton** [comment]
Fixed on the 2.18.x branch.
